src/main.rs

```diff
@@ -40,10 +40,8 @@ fn player_level_up(
 
 fn main() {
     App::new()
-        .insert_resource(WgpuSettings{backends: None,..default()})
+        .add_plugin(RenderPlugin {wgpu_settings: WgpuSettings{backends: None,..default()}})
         .insert_resource(GameState {counter: 10})
         .add_plugins(DefaultPlugins)
-        .add_stage("AI",SystemStage::parallel())
-        .add_stage("SRS",SystemStage::parallel());
```

Comment:
System stages are removed. Instead we need to use system sets. I am unsure, whether you need to explicitly specify if a System Set should run in parallel anymore. Don't they do that by default?

I would remove these calls until we actually have something to run in parallel and can test the behavior.

Unfortunately, System Sets are not documented well. The [set documentation in the bevy cookbook](https://bevy-cheatbook.github.io/programming/system-sets.html) seems outdated as well. I created an issue there: https://github.com/bevy-cheatbook/bevy-cheatbook/issues/204

I have created my own notes on system sets, if you are interested: https://github.com/Abb4/bevy-obsidian-notes/blob/main/version0.10/bevy%20System%20Sets.md#creating